hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/records/ApplicationHomeSubCluster.java

```diff
@@ -160,7 +182,8 @@ public String toString() {
         .append("ApplicationId: ").append(getApplicationId()).append(", ")
         .append("HomeSubCluster: ").append(getHomeSubCluster()).append(", ")
         .append("CreateTime: ").append(getCreateTime()).append(", ")
-        .append("]");
+        .append("ApplicationSubmissionContext: ").append(getApplicationSubmissionContext())
+        .append(", ").append("]");
```

Comment:
A little weird we have a comma at the end.
It was there before but is that intended?

Reply:
Thank you very much for the reminder! It was my oversight, and the comma at the end was not intended. I will improve this part of the code.